src/color-chart/color-chart.js

```diff
@@ -26,28 +30,40 @@ const Self = class ColorChart extends ColorElement {
 		super();
 
 		this._el = {
-			slot:   this.shadowRoot.querySelector("slot"),
+			slot:   this.shadowRoot.querySelector("slot:not([name])"),
+			channel_picker: this.shadowRoot.getElementById("channel_picker"),
 			chart:  this.shadowRoot.getElementById("chart"),
 			xTicks: this.shadowRoot.querySelector("#x_axis .ticks"),
 			yTicks: this.shadowRoot.querySelector("#y_axis .ticks"),
 			xLabel: this.shadowRoot.querySelector("#x_axis .label"),
 			yLabel: this.shadowRoot.querySelector("#y_axis .label"),
 		};
+
+		this._slots = {
+			color_channel: this.shadowRoot.querySelector("slot[name=color-channel]"),
+		};
 	}
 
 	connectedCallback () {
 		super.connectedCallback();
 		this._el.chart.addEventListener("colorschange", this, {capture: true});
+		this._slots.color_channel.addEventListener("change", this, {capture: true});
```

Comment:
This is still change, is that intentional?

Reply:
No, it's because I'm an idiot. 🤦‍♂️ Sorry about that. Fixed.